geode-core/src/main/java/org/apache/geode/distributed/internal/tcpserver/TcpHandler.java

```diff
@@ -1,30 +1,27 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
- * agreements. See the NOTICE file distributed with this work for additional information regarding
- * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance with the License. You may obtain a
- * copy of the License at
  *
- * http://www.apache.org/licenses/LICENSE-2.0
+ * * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
```

Comment:
Huh, this license file header format got messed up.

Reply:
very odd, I'll check it out...